gateway/handle_http_auth.go

```diff
@@ -234,6 +285,33 @@ func (gw *Handle) handleFailureStats(errorMessage, reqType string, arctx *gwtype
 				WorkspaceID: arctx.WorkspaceID,
 				SourceType:  arctx.SourceCategory,
 			}
+		case response.DestinationDisabled:
+			stat = gwstats.SourceStat{
+				SourceID:    arctx.SourceID,
+				WriteKey:    arctx.WriteKey,
+				ReqType:     reqType,
+				Source:      arctx.SourceTag(),
+				WorkspaceID: arctx.WorkspaceID,
+				SourceType:  arctx.SourceCategory,
+			}
+		case response.InvalidDestinationID:
+			stat = gwstats.SourceStat{
+				SourceID:    arctx.SourceID,
+				WriteKey:    arctx.WriteKey,
+				ReqType:     reqType,
+				Source:      arctx.SourceTag(),
+				WorkspaceID: arctx.WorkspaceID,
+				SourceType:  arctx.SourceCategory,
+			}
+		case response.NoDestinationIdInHeader:
+			stat = gwstats.SourceStat{
+				SourceID:    arctx.SourceID,
+				WriteKey:    arctx.WriteKey,
+				ReqType:     reqType,
+				Source:      arctx.SourceTag(),
+				WorkspaceID: arctx.WorkspaceID,
+				SourceType:  arctx.SourceCategory,
+			}
```

Comment:
Seems like duplicate code?
```suggestion
		case response.DestinationDisabled, response.InvalidDestinationID, response.DestinationNotFound
			stat = gwstats.SourceStat{
				SourceID:    arctx.SourceID,
				WriteKey:    arctx.WriteKey,
				ReqType:     reqType,
				Source:      arctx.SourceTag(),
				WorkspaceID: arctx.WorkspaceID,
				SourceType:  arctx.SourceCategory,
			}
```